palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/policy/Policy.java

```diff
@@ -104,72 +104,72 @@ private void addMessage(final String newMessage, final Rules rules) {
     public Policy<RULE_DATA_TYPE> recordLevelRule(final String message, final Rule<RULE_DATA_TYPE> rule) {
         requireNonNull(message, "The message cannot be null and should indicate what the rule is doing.");
         requireNonNull(rule, "Cannot set a null rule.");
-        Rules<RULE_DATA_TYPE> recordRules = getRecordRules();
-        recordRules.rule(generateUUID(), rule);
-        addMessage(message, recordRules);
+        Rules<RULE_DATA_TYPE> recordRule = getRecordRules();
```

Comment:
I think these are still recordRules, because it is the Rules object which is a collection of Rules which are being applied to a data type.